CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ Todos los cambios notables a este proyecto serán documentados en este archivo.
 El formato está basado en [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 y este proyecto adhiere a [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.1.0] - 2019-04-04
```

Comment:
why `1.1.0` if the previous version is `2.1.0`? sounds like a writing error 😅

Reply:
It is, I should slow down, had too much coffe. I'll fix it in the next commit